store/rootmulti/store.go

```diff
@@ -100,65 +100,121 @@ func (rs *Store) GetCommitKVStore(key types.StoreKey) types.CommitKVStore {
 	return rs.stores[key].(types.CommitKVStore)
 }
 
-// Implements CommitMultiStore.
+// LoadLatestVersionAndUpgrade implements CommitMultiStore
+func (rs *Store) LoadLatestVersionAndUpgrade(upgrades *types.StoreUpgrades) error {
+	ver := getLatestVersion(rs.db)
+	return rs.loadVersion(ver, upgrades)
+}
+
```

Comment:
Should we also have the ability to load any version with upgrades (not just the latest)?

Reply:
Sure. I just added `LoadVersionAndUpgrade` method